captain/tests/topology_test.py

```diff
@@ -0,0 +1,45 @@
+import unittest
+from queue import Queue
+from .test_apps.sample_app import graph as sample_app_graph
+from captain.models.topology import Topology
+from captain.types.worker import JobInfo
+from flojoy import JobSuccess
+
+
+class TopologyTest(unittest.TestCase):
```

Comment:
could you add a test that the end result is what was expected? i.e use numpy to compute the same values that the topology is computing and assert that they are equal